openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -1206,3 +1212,170 @@ def validate_glusterfsd_memory_usage(self, size_limit):
                     "Failed memory used  of glusterfsd {} is greater than the"
                     " expected size {} for node/pod {}".format(
                         memory_used, size_limit, gluster_node))
+
+    def configure_setup(self, max_volume_count, total_volume_count):
+        # Get revised version of heketi
+        revision = oc_get_custom_resource(
+            self.ocp_master_node[0], 'dc', ':status.latestVersion',
+            self.heketi_dc_name)
+
+        # Set env variable to support max 2000 volumes in heketi dc
+        cmd = ("oc set env dc/{} HEKETI_GLUSTER_MAX_VOLUMES_PER_CLUSTER="
```

Comment:
This option is required if we need to create volume > 1000, So I would suggest editing the dc if the  total_volume_count exceed 1000

Reply:
what if we want to scale a setup for 100 pvc, it will update the heketi
Which is not required?